packages/ui/components/profile/ProfileVoteButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  ChatAltIcon,
+  CheckIcon,
+  HandIcon,
+  XIcon,
+} from '@heroicons/react/outline'
+import clsx from 'clsx'
+import { useTranslation } from 'react-i18next'
+
+import { Button } from '../Button'
+
+export interface ProfileVoteButtonProps {
+  variant: 'abstain' | 'noWithVeto' | 'no' | 'yes'
+  pressed?: boolean
+}
+
+export const ProfileVoteButton = ({
+  variant,
+  pressed = false,
+}: ProfileVoteButtonProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <Button
+      className={clsx('pt-2 pb-6 pl-5 mb-2 w-full h-10', {
+        'border border-2 border-border-primary': pressed,
+      })}
+      contentContainerClassName={clsx('justify-between text-sm', {
+        'text-sm primary-text': !pressed,
+      })}
```

Comment:
The Button borders and positioning look weird on this, let me play around with the Button component.